docs/setup.md

```diff
@@ -0,0 +1,20 @@
+---
+sidebar_position: 2
+---
+
+
+# Meta Quest 3 Official Setup and Interaction Guide
+
+Before diving into Vitessce Link, we recommend you spend some time setting up the Meta Quest 3 and familiarizing yourselves with the hardware and basic built-in demos. Please see the below resources:
+
+General hardware setup: [Video](https://share.vidyard.com/watch/wS1pQLMmPo9jfvfi3WfzyA)	
+
+General Setup of the Meta Quest 3:  
+[Video](https://www.youtube.com/watch?v=wPQQ6vHlIN0&ab_channel=MetaQuest)
```

Comment:
This appears to be the same video that we link to on line 10, just hosted on YouTube - are these meant to be different?